hack/dependencies.yml

```diff
@@ -0,0 +1,21 @@
+---
+ytt:
+  version: v0.38.0
+  checksum_darwin: 1f84e26457a5efd2470df408072bf2f182c939414e4958f82e19e122d2ee120c
+  checksum_linux: 2ca800c561464e0b252e5ee5cacff6aa53831e65e2fb9a09cf388d764013c40d
+kbld:
+  version: v0.32.0
+  checksum_darwin: 5fc8a491327294717611974c6ab3da2bda3f3809ef3147c1e8472ac62af3ee18
```

Comment:
if you duplicated the tool name in these leaf node names (e.g. here, `kbld_checksum_darwin`) then you get to have a structured file that you can use with yq outside of the install-deps script, and it's ~straightforward for the install-deps script to use sed to scrape the desired values.    do you think that's worth it?

Reply:
I like it :) I think it would work for sure, it's both structured where it needs to be, and not where its not. Shouldn't take long to change it to this. 👍